packages/remote-config/index.ts

```diff
@@ -139,3 +146,22 @@ declare module '@firebase/app-types' {
     remoteConfig(): RemoteConfigType;
   }
 }
+/**
+ * this is a public static method provided to users that wraps two different checks:
+ *
+ * 1. check if IndexedDB is supported by the browser environment.
+ * 2. check if the current browser context is valid for using IndexedDB.
+ *
+ */
+async function isSupported(): Promise<boolean> {
```

Comment:
Thinking (no action req'd): the change to `registerRemoteConfig` will throw if IndexedDB is unavailable, in which case we'd need a method to call prior to registration, which is why `isSupported` is static, ie:

```
import * as remoteconfig from 'firebase/remote-config';
import * as firebase from 'firebase/app';

if (remoteconfig.isSupported()) {
  const rc = firebase.remoteconfig();
  ...
}
```